db/migrations/0001_initial.py

```diff
@@ -2,43 +1,58 @@
+# Generated by Django 4.0.2 on 2024-11-06 10:12
 
 from django.db import migrations, models
+import django.db.models.deletion
 
 
 class Migration(migrations.Migration):
 
     initial = True
 
-    dependencies = []
+    dependencies = [
+    ]
 
     operations = [
         migrations.CreateModel(
-            name="Actor",
+            name='Actor',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('first_name', models.CharField(max_length=255)),
+                ('last_name', models.CharField(max_length=255)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='CinemaHall',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('rows', models.IntegerField()),
+                ('seats_in_row', models.IntegerField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Genre',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Movie',
             fields=[
-                (
-                    "id",
-                    models.AutoField(
-                        auto_created=True,
-                        primary_key=True,
-                        serialize=False,
-                        verbose_name="ID",
-                    ),
-                ),
-                ("first_name", models.CharField(max_length=255)),
-                ("last_name", models.CharField(max_length=255)),
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=255)),
+                ('description', models.TextField()),
+                ('actors', models.ManyToManyField(related_name='actors', to='db.Actor')),
+                ('genres', models.ManyToManyField(related_name='genres', to='db.Genre')),
```

Comment:
The `related_name` for the `genres` field in the `Movie` model is set to 'genres', which is the same as the field name. It's recommended to use a different `related_name`, such as 'movies' or 'movie_genres', to avoid confusion and potential conflicts.